### PR TITLE
OSSMDOC-415: Document confirmation step for service mesh configuration tasks 

### PR DESCRIPTION
**Pull request by rh-tokeefe**
Make sure to include the following in the description
- Aligned team: Dev Tools
- For branches: 
- Jira: https://issues.redhat.com/browse/OSSMDOC-415
- Direct link to doc preview: https://deploy-preview-37269--osdocs.netlify.app/openshift-enterprise/latest/welcome/index.html
- SME review: @ tbd
- QE review: @yxun
- Peer review: @rolfedh 
- Cherry pick to 4.6, 4.7, 4.8, 4.9, and 4.10

### CODE REVIEW

**Review by yxun — APPROVED** [review]

File: modules/ossm-federation-create-meshPeer.adoc [hunk 1]
```diff
@@ -84,3 +84,44 @@ spec:
...
++
+If you check the `servicemeshpeer` named `red-mesh` in `green-mesh-system`, you'll find information about the same two connections from the perspective of the green mesh.
++
+When the connection between two meshes is not established, the `ServiceMeshPeer` status indicates this in the `status.discoveryStatus.inactive` field.
++
+For more information on why a connection attempt failed, inspect the istiod log, the access log of the egress gateway handling egress traffic for the peer, and the ingress gateway handling ingress traffic for the current mesh in the peer mesh.
++
+For example, if the red mesh can't connect to the green mesh, check the following logs:
+
+* istiod in red-mesh-system
```
[the remark this answers is not in the document]
Comment (rh-tokeefe) [reply modules/ossm-federation-create-meshPeer.adoc]: I was able to catch Daniel and wordsmithed this.

**Pull request merged**